src/Microsoft.AspNetCore.Sockets.Client.Http/Internal/ServerSentEventsTransport.cs

```diff
@@ -116,12 +143,15 @@ public Task StartAsync(Uri url, IDuplexPipe application, TransferFormat transfer
                         {
                             Log.ParsingSSE(_logger, input.Length);
                             var parseResult = _parser.ParseMessage(input, out consumed, out examined, out var buffer);
+                            FlushResult flushResult = default;
```

Comment:
Not something I want to change now, but I had preferred `var flushResult = default(FlushResult)` in this situation, because it's more consistent with our "var-everywhere" style. Is this your preferred way to do this?

Reply:
I actually prefer the style you mention but you get more @pranavkm points when you use newer language features.

Reply:
And something tracking Pranav Points